Jenkinsfile-deploy

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent {
+        label "master"
+    }
+    parameters {
+        choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the front end for")
+        string(name: "TO_DEPLOY", description: "The git tag, branch or commit reference to deploy, e.g. '1'")
+    }
+    stages {
+        stage("Deploy lambda") {
+            agent {
+                ecs {
+                    inheritFrom "aws"
+                    taskrole "arn:aws:iam::${env.MANAGEMENT_ACCOUNT}:role/TDRJenkinsNodeLambdaRole${params.STAGE.capitalize()}"
+                }
+            }
+            steps {
+                script {
+                    def accountNumber = getAccountNumberFromStage()
```

Comment:
`getAccountNumberFromStage` function is available in the shared library

Reply:
I've changed it.

Reply:
Looks like the stage param for the `tdr.getAccountNumberFromStage()` function that you added